Subdomain (#2225)
galaxyproject/galaxy-hub: content/news/2023-10-17-earth-system/index.md

---
title: "A brand new subdomain of Galaxy Europe : earth-system.usegalaxy.eu "
date: "2023-10-17"
authors: Marie Jossé
tease: "A subdomain to process, analyse and access Earth System and Environmental data"

> hide_tease: true
> subsites: [all-eu,global]

Reviewer: ```suggestion
tags: [esg-wp5, esg]
subsites: [all, all-eu, esg]
```

---

## Earth System ?? What's that ?

<div class='center'>
<img src="sys-earth.jpg" alt="Earth System"/>
</div>

Earth system models (ESM) seek to simulate all relevant aspects of the Earth system.
ESMs are composed by a set of equations describing atmospheric and oceanic circulation and thermodynamics, but also the biological and chemical processes that feedback on to the physics of climate, all solved for a number of locations in space that form a three-dimensional grid over the surface of the Earth and underneath the surface of the oceans.

### Fair-Ease : an EOSC project

> An approach to observation and modelling of the Earth System, Environment and Biodiversity.

Reviewer: A link to the FAIR-Ease homepage?

The project focuses onto 2 main aspects:
- Interdisciplinary Data Discovery and Access Service : for discovery and access to environmental and multidisciplinary & aggregated data
- Earth Analytical Lab and Data Lake : an easy way to visualise, analyse and process environmental data on-demand.

## Five domains creating a model of an Earth System on Galaxy

- Coastal Water Dynamics : focuses on the coastal marine environment near river estuaries, where important processes take place.
- Earth Critical Zone : monitors land and soil degradation.
- Volcano Space Observatory : monitors global volcanic activity, allowing the focus on any major volcanic eruption worldwide.
- Ocean Bio-Geochemical Observations: addresses questions regarding the health of marine ecosystems (e.g. ocean acidification, ...) and needs for ocean resource management.
- Marine Omics : analyses of spatial- and time-comparable marine microbial metagenomics data sets for the exploration of biodiversity and its correlations with environmental quality.

### Tools, tools and more tools

> Thanks to the help and partnership of Galaxy Europe and EuroScienceGateway project we were able to implement 6 interactive tools on usegalaxy.eu. There are now also available on [earth-system.usegalaxy.eu](https://earth-system.usegalaxy.eu/). Go running on the subdomain page to try out these tools !

Reviewer: link to eurosciencegateway.eu?

Reviewer: ```suggestion
Thanks to the help and partnership of Galaxy Europe and [EuroScienceGateway](https://eurosciencegateway.eu) project we were able to implement 6 interactive tools on usegalaxy.eu. They are now also available on [earth-system.usegalaxy.eu](https://earth-system.usegalaxy.eu/)! Go run them on [earth-system.usegalaxy.eu](https://earth-system.usegalaxy.eu/) and try these tools!
```

Reviewer: Maybe a list of those tools? With links?

> Don't hesitate to give use some feedbacks and help if you think they should be improve. As said before they are quite new and need to be tested and approved.

Reviewer: ```suggestion
Don't hesitate to give us some feedback and help if you think they should be improved. As said before they are quite new and need to be tested and vetted by the community.
```

> The final goal will be to have at least one workflow of tools to access, process and visualise data for each domain cited above. Hopefully even more than that ! 

Reviewer: ```suggestion
One goal of FAIR-Ease is to have at least one workflow of tools to access, process, and visualise data for each domain cited above. Hopefully even more than that! 
```

> If you are interrested by earth system and environmental sciences and have tools that you think could fit right into earth-system.usegalaxy.eu, please contact us so we can try to work together on them ! The more the merrier ! 

Reviewer: ```suggestion
If you are interested in earth-system and environmental sciences and have tools that you think could fit right into [earth-system.usegalaxy.eu](https://earth-system.usegalaxy.eu/), please contact us so we can try to work together! The more the merrier! 
```

> This work takes place within the cooperation of the Fair-Ease and EuroScienceGateway projects (to know more on the projects see the [blog](https://galaxyproject.org/news/2023-05-21-fair-ease-euro-science-gateway/)).

Reviewer: ```suggestion
This work takes place within the cooperation of the FAIR-Ease and EuroScienceGateway projects (to know more on the projects see the [blog](https://galaxyproject.org/news/2023-05-21-fair-ease-euro-science-gateway/)).
```
